### PR TITLE
Added Keras Model and Predict on Bot-end

### LINKED ISSUE #7: Build Classifier Model
- [x] Build Spam/Ham Classifier Model
- [x] Decide on passing threshold for misclassification rate
- [x] Test for Accuracy / Improve, if poor

### PR DESCRIPTION
**Pull request by ajwad-shaikh**
Fixes #7 

- Created new Keras Model
  - Binary Classifier Neural Network using Sequential Layers (5-16-16-1) 
  - Sigmoid activator at the final stage to give binary output (almost)
- Imported TF.js model into our bot
- Predicting tensors on bot-end

### CODE REVIEW

**Review by vrushti-mody — APPROVED** [review]
Looks really good
